### ISSUE TITLE
В релизных архивах отсутствует содержимое libprov

### ISSUE THREAD

**Issue by vaminakov** [gost-engine/engine#456]
В сформированных релизах (tar.gz, zip) последних версий отсутствует содержимое папки libprov, что делает невозможной сборку в соответствии с инструкцией. 
Для успешной сборки приходится отдельно скачивать содержимое из main ветки git.
Возможно ли устранить данную проблему и формировать релизы со всеми необходимыми для сборки зависимостями? Заранее спасибо!

**Comment by vt-alt** [comment]
Если вы про страницу https://github.com/gost-engine/engine/releases то в Assets там файлы которые создает GitHub автоматически и это всего лишь содержимое команды git archive.

**Comment by vaminakov** [comment]
@vt-alt Да, про нее. Если взять архив из релиза, то из него невозможно скомпилировать библиотеку, в отличие от клонирования репозитория.
